### DISCUSSION TITLE
How to set StudyName in Zombie Fish Component

### DISCUSSION THREAD

**Discussion by shasanc41** [Tunny-gh/Tunny_in_Open_Sea#248] [General]
Hi hiroaki,

I have been trying to use the Zombie fish component. I come across the error every time I try to open the sample file (optimize_without_ui), followed by automatically closing the Rhino.
![image](https://github.com/hrntsm/Tunny/assets/87524515/cfe21712-afd8-4974-98da-57ad8210405e)

It is asking to change the name from Test 1. I opened the sample file in locked mode and tried to find if I can update the name of the test. I am not sure where am expected to update the name.

Thanks,
Shasan

**Comment by hrntsm** [comment]
Hi @shasanc41 

Thank you for your feedback!
The BoneFish component is now a very expert-oriented feature. Instead of eliminating the UI, it uses optimization settings stored in a json file.
The json file can be opened by right-clicking on the component. However, it is difficult to understand how this json is specified because it is not yet documented.

The easiest way to set value to json is use Tunny's UI. The json will reflect the settings when the UI is closed for the last time.
In this case, there are two solutions.

1.  if StudyName is left empty, Tunny will automatically assign a unique name to the study so that there are no duplicate names. (However, this will not allow continuation.
2. If you want to continue, check the Continue checkbox. The value will be reflected in the json and you can continue optimization with the StudyName you set.

![image](https://github.com/hrntsm/Tunny/assets/23289252/40d4d91f-5aae-4de6-b7ed-41c1f005c251)
<img width="912" alt="Screenshot 2024-02-20 at 9 17 28" src="https://github.com/hrntsm/Tunny/assets/23289252/c4ac385d-4d31-414c-a3f4-9eb0076fc382">

BoneFish is a experimental feature. Please let me know if you have an opinion on how it could be made more user-friendly.
